src/NServiceBus.Core/Sagas/SagaIdGeneratorContext.cs

```diff
@@ -5,7 +5,7 @@ namespace NServiceBus.Sagas
     /// <summary>
     /// Context provided to the saga id generator.
     /// </summary>
-    public class SagaIdGeneratorContext : ContextBag
+    public class SagaIdGeneratorContext : IExtendable
```

Comment:
Interesting, we haven't done this with other context. Should we align others or remove the interface here?

https://github.com/Particular/NServiceBus/blob/develop/src/NServiceBus.Core/Routing/DistributionContext.cs#L49